docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -5,6 +5,8 @@ author: nkolev92
 ms.author: nikolev
 ms.date: 03/15/2022
 ms.topic: conceptual
+f1_keywords: 
+  - "vs.toolsoptionspages.nuget_package_manager.package_source_mapping"

```

Comment:
Can you help me understand how this works?

Reply:
I'm not sure specifically what you want to know, but basically VS Options has a help function, and that opens MSDN docs to a particular page. I'm associating the VS Options keyword (the options page name) to this page, so that F1 or `?` button launches this docs page.

Reply:
that's helpful, thanks.